.github/workflows/npm-publish.yml

```diff
@@ -8,23 +7,25 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        version: [12, 16]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Setup Node.js
-        uses: actions/setup-node@v3
+      - name: Setup Node.js v12
+        uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.version }}
+          node-version: 12
 
       - name: Install dependencies
         run: npm ci
 
+      - name: Setup Node.js v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
```

Comment:
move it after Build step

Reply:
First of it's generally a good practice to use the same Node.js version for both building and releasing as it, apart from adding unnecessary overhead to the workflow, also reduces the risk of compatibility issues between the build environment and the release environment. 

I came to realization that there will be compatibility issues with this approach between the different Node.js versions that could cause problems during the build or release process as some dependencies may not work correctly with one version of Node.js but work fine with another, an example of which in our case would be `node-sass`.